slick.grid.js

```diff
@@ -1860,6 +1860,20 @@ if (typeof Slick === "undefined") {
       stringArray.push("</div>");
     }
 
+    function htmlEscape(inputString) {
```

Comment:
It would be great to add tests for this

Reply:
this is copied from https://github.com/microsoft/azuredatastudio/blob/main/src/sql/base/common/strings.ts#L10

Reply:
Doesn't really mean we shouldn't add tests...

Reply:
I meant to say: please also add a comment explaining where this is coming from.

Reply:
Test has been added to grid.js